odoo/addons/test_new_api/tests/test_new_fields.py

```diff
@@ -1280,6 +1280,134 @@ def test_27_company_dependent(self):
         self.assertEqual(attribute_record.company.foo, 'DEF')
         self.assertEqual(attribute_record.bar, 'DEFDEF')
 
+    def test_28_company_dependent_search(self):
+        """ Test the search on company-dependent fields in all corner cases.
+            This assumes that get_multi/set_multi are correct, and that
+            filtered_domain() correctly mimics the search.
```

Comment:
`get_multi` seems unused, and it seems like it'd be clearer to state the assumption as "filtered_domain() implements the correct behaviour"? That is both clearer and providing more leeway.

Reply:
`_get_multi` is not used directly, but used by the `filtered_domain`. If the `_get_multi`/`_set_multi` has the same issue as the `search_multi` this test won't detect the issue. In other words, it tests the consistence between these `_get_multi`/`_set_multi`/`search_multi`.